_template_python/INDICATOR_DEV_GUIDE.md

```diff
@@ -260,6 +269,7 @@ In an ideal case, the data exists at one of our [already covered geos](https://c
 * ZIP
 * MSA (metro statistical area, int)
 * HRR (hospital referral region, int)
+* Nation
```

Comment:
Considering Roni just asked us to add hhs geo level to an indicator, I think we should add it to the list:
```
* HHS (HHS regions: https://www.hhs.gov/about/agencies/iea/regional-offices/index.html)
```